src/components/Settings/Settings.js

```diff
@@ -38,61 +40,44 @@ export default class Settings extends React.Component {
             value={this.props.currentNetwork}
             onChange={this.handleChangeSelectedNetwork}
           >
-            <option value="MainNet">MainNet</option>
-            <option value="TestNet">TestNet</option>
-            <option value="CozNet">CozNet</option>
-            <option value="nOSLocal">nOS Local</option>
-
-            {
-              this.props.allNetworks.map((network) => {
-                return (
-                  <option
-                    key={network.name}
-                    value={network.neoscan} // TODO: Change this to use name
-                  >
-                    {network.name}
-                  </option>
-                );
-              })
-            }
+            {map(settings.networks, this.renderNetworkOption)}
           </select>
         </label>
-        <div className={styles.networkDetails}>
-          Custom Neoscan URL: {this.props.currentNetwork}
```

Comment:
Would it be usefull to show the url neoscan?

Reply:
Seems like this is duplicating the current network, which we already show in the status bar / footer of the app.  Am I misunderstanding?

Reply:
Ahhh, okay I see it now, yeah in that case then theres no need to show it again.